projects/packages/my-jetpack/src/products/class-product.php

```diff
@@ -472,19 +496,19 @@ public static function get_status() {
 			// However if the standalone plugin for this product is active, then we will defer to showing errors that prevent the module from being active
 			// This is because if a standalone plugin is installed, we expect the product to not show as "inactive" on My Jetpack
 			if ( static::$requires_plan || ( ! static::has_any_plan_for_product() && static::$has_standalone_plugin && ! self::is_plugin_active() ) ) {
-				$status = static::$has_free_offering ? Products::STATUS_NEEDS_PURCHASE_OR_FREE : Products::STATUS_NEEDS_PURCHASE;
+				$status = static::is_owned() && static::$has_free_offering && ! static::$requires_plan ? Products::STATUS_NEEDS_ACTIVATION : Products::STATUS_NEEDS_PURCHASE;
```

Comment:
So in this case, I suppose "NEEDS_PURCHASE" will mean it needs a plan of some kind, even if that purchase is $0?

It would be good to document what we mean by "NEEDS_PURCHASE" so folks don't assume it's only paid stuff. 

We could refactor it to be "NEEDS_PLAN" or something else more descriptive in a later iterations 🤔

Reply:
Yes that is correct, and I think you're right. NEEDS_PLAN would probably be a better name for this. I'll go ahead and update that in this PR since I'm already updating one of the status names